set_theia_version.sh

```diff
@@ -77,28 +77,38 @@ change_change_che_theia_init_baranch () {
 
 # first is path to dir which contains 'package.json', second is Theia version
 updateTheiaDependencies() {
-    sed -i -r -e 's/("@theia\/..*": )(".*")/\1"'$2'"/' ./$1/package.json
+    sed -i -r -e '/plugin-packager/!s/("@theia\/..*": )(".*")/\1"'$2'"/' ./$1/package.json
+}
+
+# set version attribute in 'package.json', first path to dir with 'package.json', secod the new version
+updateVersion() {
+    sed -i -r -e 's/("version": )(".*")/\1"'$2'"/' ./$1/package.json
+}
+
+# first is path to dir which contains 'package.json', second is che-Theia version
+updateChePluginDependencies() {
+    sed -i -r -e '/@eclipse-che\/api|@eclipse-che\/workspace-client/!s/("@eclipse-che\/..*": )(".*")/\1"'$2'"/' ./$1/package.json
 }
 
 
-ask_for_change_che_theia_init_baranch
+# ask_for_change_che_theia_init_baranch
 
 read -p "Enter Theia version : "  theiaVersion
 
-sed -i -e "s/RUN git clone -b 'master'/RUN git clone -b 'v${theiaVersion}'/" ./dockerfiles/theia-dev/e2e/Dockerfile
-sed -i -e "s/ARG THEIA_VERSION=..*/ARG THEIA_VERSION=${theiaVersion}" ./dockerfiles/theia/Dockerfile
+# sed -i -e "s/RUN git clone -b 'master'/RUN git clone -b 'v${theiaVersion}'/" ./dockerfiles/theia-dev/e2e/Dockerfile
+# sed -i -e "s/ARG THEIA_VERSION=..*/ARG THEIA_VERSION=${theiaVersion}/" ./dockerfiles/theia/Dockerfile
 
-for dir in extensions/*
-do
-    echo ${dir}
-    updateTheiaDependencies ${dir} ${theiaVersion}
-done
+# for dir in extensions/*
```

Comment:
if commented, just delete it no ?
it's still in git history

Reply:
Oh, no, I forgot to uncomment that code